integration_tests/network.py

```
@@ -179,40 +182,50 @@ def setup_custom_mantra(
+                "pystarport",
+                "init",
+                "--config",
+                config,
+                "--data",
+                path,
+                "--base_port",
+                str(base_port),
+                "--no_remove",
+            ]
+            if relayer == cluster.Relayer.RLY.value:
+                cmd = cmd + ["--relayer", str(relayer)]
```

Ruff (0.12.2)

Consider `[*cmd, "--relayer", str(relayer)]` instead of concatenation

Replace with `[*cmd, "--relayer", str(relayer)]`

(RUF005)